templates/views/email-password-interaction/main.css

```diff
@@ -0,0 +1,2948 @@
+/* scss/Variables/_Variables.scss */
```

Comment:
Is all of this ours or generated? License?

Reply:
It's ripped from the Inrupt style guide.